applications/crossbar/src/crossbar_doc.erl

```diff
@@ -284,15 +284,15 @@ load_view(View, Options, Context, StartKey, PageSize) ->
     load_view(View, Options, Context, StartKey, PageSize, 'undefined').
 
 load_view(View, Options, Context, StartKey, PageSize, FilterFun) ->
+    Dbs = props:get_value('databases', Options, [cb_context:account_db(Context)]),
+    ExistingDbs = lists:filter(fun couch_mgr:db_exists/1, Dbs),
```

Comment:
When `couch_mgr:get_results/3` is actually called below, the code now branches on `{'error', 'not_found'}` and continues to process the view for the next DB (if any). I think this filter is no longer necessary. Thoughts?